Makefile

```diff
@@ -11,6 +11,7 @@ ensure::
 publish_packages:
 	$(call STEP_MESSAGE)
 	./scripts/publish_packages.sh
+	$$(go env GOPATH)/src/github.com/pulumi/scripts/ci/build-package-docs.sh ${PROJECT_NAME}
```

Comment:
for my own knowledge.. why are there two $$ here?

Reply:
Single dollar sign is for a variable defined in the Makefile. Double dollar sign is for a shell variable.